src/mem_table.rs

```diff
@@ -44,17 +44,20 @@ pub(crate) struct ValueWithAttributes {
 }
 
 impl<'a> KeyValueIterator for MemTableIterator<'a> {
-    async fn next_entry(&mut self) -> Result<Option<KeyValueDeletable>, SlateDBError> {
+    async fn next_entry(&mut self) -> Result<Option<RowEntry>, SlateDBError> {
         Ok(self.next_entry_sync())
     }
 }
 
 impl<'a> MemTableIterator<'a> {
-    pub(crate) fn next_entry_sync(&mut self) -> Option<KeyValueDeletable> {
-        self.0.next().map(|entry| KeyValueDeletable {
+    pub(crate) fn next_entry_sync(&mut self) -> Option<RowEntry> {
+        self.0.next().map(|entry| RowEntry {
             key: entry.key().clone(),
             value: entry.value().value.clone(),
-            attributes: entry.value().attrs.clone(),
+            flags: Default::default(),
+            seq: 0,
```

Comment:
thoughts on splitting this PR into two? the first can make the changes around encoding (removing `RowFeatures` from the SST) and the second introduces `seq` to the codec.

Reply:
Nevermind, I don't think this actually introduced too much more code so let's just keep it as is for now